server/middleware/ensure-logged-in.js

```diff
@@ -0,0 +1,24 @@
+var loopback = require('loopback');
+
+module.exports = function ensureLoggedIn(options) {
```

Comment:
Had to re-implement the default `connect-ensure-logged-in` middleware because it doesn't pick up the user even if there is a valid access token.